### PR TITLE
add a vizhub section to vizzes` package.json

### PR DESCRIPTION
**Pull request by stushurik**
- allow to override an imported package global name
- allow to set path to umd a bundle of an imported package
- take packages from the npm in a "exported" vizz installation

### CODE REVIEW

File: packages/presenters/src/bundle.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ import vizhubLibraries from 'vizhub-libraries';
 import bubleJSXOnly from './bubleJSXOnly';
 import svelte from './svelte';
 import hypothetical from './hypothetical';
+import { vizhubLibraries as userLibrariesSettings } from './packageJson';
```
Comment (curran) [inline packages/presenters/src/bundle.js]: I find renaming on import generally confusing, as the variable names don't match between the various files.

**Review by curran — APPROVED** [review]
Looks great! Just minor stylistic nits I can fix.

Will put into staging and QA.

File: packages/presenters/src/packageJson.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+export const dependencies = files => packageJSON(files).dependencies;
+
+export const vizhubLibraries = files => {
+  const vizhubConfig = packageJSON(files).vizhub;
+  return vizhubConfig ? vizhubConfig.libraries : {}
+}
+
+export const dependencySource = ({name, version}, libraries) => {
+  const path = libraries[pkg] ? libraries[pkg].path || '' : '';
```
Comment (curran) [inline packages/presenters/src/packageJson.js]: `pkg` is undefined here. Should be `name`?

Comment (stushurik) [reply packages/presenters/src/packageJson.js]: yes

**Comment by curran** [comment]
Tested in staging. Behaves nicely! Will merge.

**Pull request merged**